<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WCUI_KDZ</class>
 <widget class="QWidget" name="WCUI_KDZ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>微信</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>700</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QLabel" name="label_WC">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>200</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>微信</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_WCRW">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>125</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_WCID">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>205</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_WCpw">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>345</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_WCID">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>420</width>
       <height>70</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_WCpw">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>340</y>
       <width>420</width>
       <height>70</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_WCL">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>510</y>
       <width>190</width>
       <height>85</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_WCR">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>510</y>
       <width>190</width>
       <height>85</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_WCregister">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>180</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_WCIDnew">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_WCnamenew">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>215</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_WCpwnew">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>315</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_WCpwagainnew">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>413</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_WCIDnew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_WCnamenew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>210</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入昵称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_WCpwnew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_WCpwagainnew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>410</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请确认密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_WCemailnew">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>515</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>邮箱</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_WCemailnew">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>510</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入邮箱</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_WCregister">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>600</y>
       <width>720</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_RWC">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>125</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
